ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -589,177 +672,96 @@ def test_repeat(
 
 
 @st.composite
-def _tile_helper(draw):
-    dtype, value, shape = draw(
-        helpers.dtype_and_values(
-            available_dtypes=ivy_np.valid_dtypes, ret_shape=True, min_num_dims=1
-        )
-    )
-    reps = draw(
-        helpers.dtype_and_values(
-            available_dtypes=(ivy_np.int8, ivy_np.int16, ivy_np.int32, ivy_np.int64),
-            shape=(len(shape),),
-            min_value=0,
-            max_value=10,
+def _split_helper(draw):
+    noss_is_int = draw(
+        st.shared(helpers.ints(min_value=1, max_value=2), key="noss_type").map(
+            lambda x: x == 1
         )
     )
-    return (dtype, value), reps
-
-
-# tile
-@given(
-    dtype_value_repeat=_tile_helper(),
-    as_variable=helpers.array_bools(num_arrays=2),
-    with_out=st.booleans(),
-    num_positional_args=helpers.num_positional_args(fn_name="tile"),
-    native_array=helpers.array_bools(num_arrays=2),
-    container=helpers.array_bools(num_arrays=2),
-    instance_method=st.booleans(),
-    data=st.data(),
-)
-@handle_cmd_line_args
-def test_tile(
-    *,
-    data,
-    dtype_value_repeat,
-    as_variable,
-    with_out,
-    num_positional_args,
-    native_array,
-    container,
-    instance_method,
-    fw,
-):
-
-    dtype_value, repeat = dtype_value_repeat
-
-    dtype, value = dtype_value
-    value = np.asarray(value, dtype=dtype)
-
-    repeat_dtype, repeat_list = repeat
-    repeat = np.asarray(repeat_list, dtype=repeat_dtype)
-    dtype = [dtype, repeat_dtype]
-
-    helpers.test_function(
-        input_dtypes=dtype,
-        as_variable_flags=as_variable,
-        with_out=with_out,
-        num_positional_args=num_positional_args,
-        native_array_flags=native_array,
-        container_flags=container,
-        instance_method=instance_method,
-        fw=fw,
-        fn_name="tile",
-        x=value,
-        reps=repeat,
+    shape = draw(st.shared(helpers.get_shape(min_num_dims=1), key="value_shape"))
+    axis = draw(
+        st.shared(helpers.get_axis(shape=shape, force_int=True), key="target_axis")
     )
 
+    if noss_is_int:
```

Comment:
add comments explaining this

Reply:
Explained what is being generated if `noss` is an integer in a comment